docker/maintenance/cronjob-template.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {name}
```

Comment:
Let's prefix the name on cronjob (maybe with `mediawiki-`?) so it's easy to distinguish them from other Kubernetes jobs.